package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "repository": "https://www.github.com/kairstenfay/yang-bot",
+  "license": "MIT",
```

Comment:
also took a stab at this for you but LMK if you prefer a different one!

Reply:
This is wonderful, thank you.